demo/example.md

```diff
@@ -0,0 +1,15 @@
+## Markdown Renderer
+
+Example:
+
+```html
+<paper-toolbar>
+  <paper-icon-button icon="menu"></paper-icon-button>
+  <div class="title">Title</div>
+  <paper-icon-button icon="more"></paper-icon-button>
+</paper-toolbar>
+```
+
+_Nifty_ features.
+
+![Polymer](polymer.svg)
```

Comment:
Could optionally just remove the image from the demo. The `polymer.svg` is 12.1kb